crates/neon-runtime/src/napi/call.rs

```diff
@@ -28,7 +29,17 @@ pub unsafe extern "C" fn get_isolate(_info: FunctionCallbackInfo) -> Env { unimp
 // FIXME: Remove. This will never be implemented
 pub unsafe extern "C" fn current_isolate() -> Env { panic!("current_isolate won't be implemented in n-api") }
 
-pub unsafe extern "C" fn is_construct(_info: FunctionCallbackInfo) -> bool { unimplemented!() }
+pub unsafe extern "C" fn is_construct(env: Env, info: FunctionCallbackInfo) -> bool {
+    let mut target: MaybeUninit<napi::napi_value> = MaybeUninit::zeroed();
+    let status = napi::napi_get_new_target(
+        env,
+        info,
+        target.as_mut_ptr()
+    );
+    assert_eq!(status, napi::napi_status::napi_ok);
+    let target: napi::napi_value = target.assume_init();
```

Comment:
I think this could use a comment. Maybe it's just me, but I had to think a LOT about whether it's valid to call `assume_init` here when the value could be `null`.

It _is_ valid and correct because `T` is a raw pointer (not a ref) and it's valid for a raw pointer to be `null`.